.github/workflows/integration.yml

```diff
@@ -0,0 +1,50 @@
+name: continuous-integration
+
+on: 
+  push:
+    branches:
+      - dev
+      - master
+    paths:
+      - 'src/**'
+      # - 'test/**'
+      - 'build-libs.sh'
+      - 'Dockerfile'
+      - 'Makefile.essentiajs'
+      - 'rollup*.js'
+      - 'tsconfig.json'
+      - '!docs/**'
+      - '!examples/**'
+  pull_request:
+    branches:
+      - dev
+      - master
+    paths:
+      - 'src/**'
+      # - 'test/**'
+      - 'build-libs.sh'
+      - 'Dockerfile'
+      - 'Makefile.essentiajs'
+      - 'rollup*.js'
+      - 'tsconfig.json'
+      - '!docs/**'
+      - '!examples/**'
+    types:
+      - opened
+      - edited
+      - reopened
+
+jobs: 
+  build:
+    runs-on: ubuntu-latest
+    container:
+      image: docker://mtgupf/essentia-emscripten:latest
+      volumes: 
+        - ${{ env.GITHUB_WORKSPACE }}:/essentia/
+      options: --rm
+    steps:
+      - uses: actions/checkout@v2
+      # - uses: kurocha/setup-cpp@v1 # necessary?
+      - run: /essentia/build-libs.sh
```

Comment:
in future we can split this bash script to run build and tests separately. Not important in this PR.

Reply:
agree, tests should eventually be a separate job, although in that case we need to find out how to share data between jobs (we will need the output of the build job for the test job)